impl/src/main/java/org/ehcache/internal/store/tiering/CacheStore.java

```diff
@@ -312,6 +313,12 @@ public void disableStoreEventNotifications() {
     }
   }
 
+  @Override
+  public CacheConfigurationChangeListener getConfigurationListener() {
+//    throw new UnsupportedOperationException("TODO Implement me!");
+    return null;
```

Comment:
As decided, the scope of this commit only permits updating onheap. However the exceptions on other cases will be removed.